src/store/servers.ts

```diff
@@ -13,28 +13,37 @@ export const servers: IServer[] = [
   {
     name     : 'Piplup (Main)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242' },
```

Comment:
I'd like to keep the IP addresses here in case that DNS doesn't work. If the IP via DNS changes it will show the new and the old one though till it changes here (as it is now actually, because you changed the IP of `smoo.raychu.xyz` from `143.198.241.242` to `35.228.67.224` causing the server now to show as offline).